src/core/server/parseJSONText.ts

```diff
@@ -1,10 +1,12 @@
+import { EditorJSObject, JSONString } from "@/app/types";
```

Comment:
This is unrelated but was failing when I ran `npm run build` locally, so I opted to fix this so we could do the typechecking on the project locally